hindsight/io_modules/derived_stream/redshift/psv.lua

```diff
@@ -13,13 +13,13 @@ local string    = require "string"
 
 setfenv(1, M) -- Remove external access to contain everything in the module
 
-local esc_chars = { ["|"] = "\\|", ["\r"] = "\\r", ["\n"] = "\\n" }
+local esc_chars = { ["|"] = "\\|", ["\r"] = "\\r", ["\n"] = "\\n", ["\0"] = "0" }
```

Comment:
Can we replace it with a literal `\0`? (2 chars, backslash, then zero).  It'd be nice to distinguish these from legitimate zeroes.

I would also be OK with truncating the string at the null char, or discarding the string as garbage (as long as we log some of them so we can debug and validate upstream if needed).